src/components/UrqlSubscription.re

```diff
@@ -15,50 +14,46 @@ type subscriptionRenderProps('a) = {
   response: UrqlTypes.response('a),
 };
 
+type handler('a, 'b) =
+  (~prevSubscriptions: option('a), ~subscription: 'b) => 'a;
+
+module SubscriptionJs = {
+  [@bs.module "urql"] [@react.component]
+  external make:
+    (
+      ~query: string,
+      ~variables: option(Js.Json.t),
```

Comment:
When you declare a named argument with type `option('a)`, it does not mean the argument is optional to the caller, it means it receives a `option('a)` type.
I believe what you wanted was: `~variables: Js.Json.t=?`
```suggestion
      ~variables: Js.Json.t=?,
```